stage1_fly/run/main.go

```diff
@@ -333,8 +338,8 @@ func stage1() int {
 		return 1
 	}
 
-	if err := os.Chdir("/"); err != nil {
-		log.PrintE("can't change to root new directory", err)
+	if err := os.Chdir(workDir); err != nil {
```

Comment:
Should we check that `workDir` does not contain `../../../` so `rkt fly` does not immediately escape the chroot?
```
os.Chdir(path.Join("/", workDir))
```

OTOH, it might not worth the protection since `rkt fly` is privileged already...